role_based_access_AG_account_multiple_monitoring_roles_template.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Role-based access for Managed and SaaS deployments with 
+  Environment ActiveGate
+Parameters:
+  ActiveGateRoleName:
+    Type: String
+    Description: IAM role name for the account hosting the ActiveGate for
+      monitoring. This must be the same name as the ActiveGateRoleName parameter 
+      used in the template for the monitored account.
+    Default: Dynatrace_ActiveGate_role
+  AssumePolicyName:
+    Type: String
+    Description: IAM policy name attached to the role for the account hosting the
+      ActiveGate
+    Default: Dynatrace_assume_policy
+  MonitoringRoleARNs:
+    Type: CommaDelimitedList
+    Description: List of ARNs for IAM roles that Dynatrace should use to get monitoring data.
```

Comment:
I'd add at the end of the description: 
`Example: arn:aws:iam::111111111:role/Dynatrace_Monitoring_Role, arn:aws:iam::222222222:role/Dynatrace_Monitoring_Role`